### ISSUE TITLE
Merge RNA polymerase II transcription cofactor activity transcription cofactor activity in parent

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#15665]
* GO:0001104 RNA polymerase II transcription cofactor activity: only used in metazoa
merge into: GO:0003712 transcription cofactor activity

* GO:0001105 RNA polymerase II transcription coactivator activity: only used in eukaroytes
 merge into:  GO:0003713 transcription coactivator activity

* GO:0001106 RNA polymerase II transcription corepressor activity: only used in metazoa
 merge into:  GO:0003714 transcription corepressor activity 

So this has no impact on annotations. 

@krchristie @ValWood @RLovering is this OK ? 

Thanks, Pascale

**Comment by RLovering** [comment]
Im happy with this

Ruth

**Comment by krchristie** [comment]
So, if you eliminate all of the RNAP II specific terms, how are people going to be able to identify the ones that affect RNAP II, versus some other RNAP? Especially considering that most people only want to know about RNAP II...

**Comment by pgaudet** [comment]
Hi @krchristie 

There are no other children terms (non-polII). Do you know of examples ? 

Pascale

**Comment by krchristie** [comment]
The issue isn't eukaryotic RNA pols I or III. However, cofactors might apply to plant pols IV and V since these are more close related to RNA pol II.

Also, the generic terms are all flagged as being in the prokaryotic subset and I have vague recollections that the bacterial transcription community may indeed refer to transcription coactivators and corepressors. Probably good to get input from @dsiegele and @jimhu-tamu before changing these terms.

**Comment by ValWood** [comment]
> So, if you eliminate all of the RNAP II specific terms, how are people going to be able to identify the ones that affect RNAP II, versus some other RNAP? Especially considering that most people only want to know about RNAP II...

Maybe this will become a documentation issue:
Retrieve all genes involved in   "transcription by RNA polymerase II (GO:0006366)"
and then break down by function (GSTF, cofactor, general).
This is kind of what @vanaukenk  was saying on the call (I think?)  were you?

But it could be a moot issue for species where "all" annotations to  "GO:0003712 transcription cofactor activity" are already pol II.

**Comment by dsiegele** [comment]
Should I start a new issue? 

I am OK with the recommended mergers, but I have a question about the definitions of:
GO:0003712 transcription cofactor activity
GO:0003713 transcription coactivator activity
GO:0003714 transcription corepressor activity

The definitions of all 3 terms include the statement "Cofactors generally do not bind the template nucleic acid, but rather mediate protein-protein interactions."  In prokaryotes, most transcription transcriptional activators and repressors do bind to the genomic nucleic acid and altering the DNA binding sites for the proteins prevents them acting.  

Also, at least in E. coli, the terms corepressor and coactivator often refer to the small molecules that modulate the activity of a protein factor.  For example, tryptophan is the corepressor for the TrpR protein: the complex binds to the genomic DNA and represses transcription of the trp operon. 
Could something be added to the comments to point out this difference.

I apologize for the late response, but it is near the end of the semester and all my neurons are occupied with grading exams.

**Comment by pgaudet** [comment]
Hi @dsiegele 

Thanks for your response. In the new organisation this would be considered a 'repressor', not a co-repressor. I understand that the bacterial nomenclature might be different; but in principle does this make sense to you ? 
GO:0001217 bacterial-type RNA polymerase transcriptional repressor activity, sequence-specific DNA binding

Yesterday I also added a taxon constraint (never in bacteria) for transcription cofactor activity (there was a single annotation to P69348/yoeB, which I have discussed with the SP curator who made it; she was waiting for more guidance to move it). 

Does that sound OK to you ? Will you be at the GOC meeting ? 

Thanks, Pascale
